torch/fx/experimental/symbolic_shapes.py

```diff
@@ -175,7 +175,7 @@ def wrap_float(self, num):
         return SymNode(sympy.Float(num), self.shape_env, float, constant=num)
 
     def clone(self):
-        return SymNode(self.expr, self.shape_env, self.pytype, constant=self.constant)
+        return self
```

Comment:
why do we want to return self here?

Reply:
We're disabling the SymNode cloning. A better refactor would be to eliminate the clone method entirely

Reply:
I see. It might be nice to add a warning here since this behavior seems a little unexpected

Reply:
Until Brian removes calls to clone from C++, this method will get (uselessly) called a bunch, warning would be spamming